spring-boot/admin/src/main/resources/application-github.yml

```diff
@@ -0,0 +1,24 @@
+spring:
+  security:
+    oauth2:
+      client:
+        registration:
+          # the following oauth client configuration is just for development testing purposes, do NOT use in a production setting
+          github:
+            client-id: e188a243cc57ef90f2b5
+            client-secret: a5630e2f873890bdc8366aac0155e8113ece0997
+            scope[0]: user:email
+            scope[1]: read:user
+            redirect-uri: http://localhost:8080/login/oauth2/code/github
+        provider:
+          github:
+            # Use the "login" attribute of the Github user info as authentication name in Spring Security
+            user-name-attribute: login
+          resource:
+            userInfoUri: https://api.github.com/user
+
+# Attribute names to be retrieved as firstName, username and company name for the CoatRack registration
+oauth2.attributes.login.name: login
```

Comment:
My assumption is that these attributes are defined by us, but not defined by Spring or any libraries we use, right? If that assumption is correct, please add a prefix `ygg.admin` to all of them to make clear that these are custom to CoatRack.

Reply:
Done